app/models/concerns/api_model_convertable.rb

```diff
@@ -0,0 +1,19 @@
+module ApiModelConvertable
```

Comment:
This totally doesn't matter, but isn't it convertible? 😅

Reply:
So it is... 😑